checker.cpp

```diff
@@ -1,17 +1,97 @@
+// ConsoleApplication3.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+
+#include <iostream>
+
 #include <assert.h>
 
-bool vitalsAreOk(float bpm, float spo2, float respRate) {
-  if(bpm < 70 || bpm > 150) {
-    return false;
-  } else if(spo2 < 90) {
-    return false;
-  } else if(respRate < 30 || respRate > 95) {
-    return false;
-  }
-  return true;
+using namespace std;
+
+float MinBPM[] = { 70 };
```

Comment:
These are arrays, but have only one value. Are they arrays because in future you want to add more minimums and maximums? In that case, rename the arrays accordingly.